player.rb

```diff
@@ -0,0 +1,30 @@
+# Class for a blackjack player to keep track of cards in hand and total value
+class Player
+  attr_reader :hand, :total
+  attr_accessor :hand, :total
+
+  def initialize
+    @hand = []
+    @total = calculate_total
+  end
+
+  def get_card_val(sum, card)
+    # Face cards count as 10
+    if %w(J K Q).include?(card.val)
+      10
+    # Ace should count as 1 or 11
+    elsif card.val == 'A' && sum + 11 >= 21
```

Comment:
I played the game a little. looks nice. I think I found a bug though. I saw a hand of:
```
A ♠
4 ♥
J ♥
```
which busted as 25. although it should be 15. problem was the `A` was seen as 11 because it was first where if it were last it would be seen as 1.

I also got

```
J ♦
A ♣
```

which should be blackjack, but I lost to dealer holding 18 as the `A` was seen as 1

just something to think about.